nebula-exchange_spark_3.0/src/main/scala/com/vesoft/nebula/exchange/reader/ServerBaseReader.scala

```diff
@@ -78,6 +69,30 @@ class MySQLReader(override val session: SparkSession, mysqlConfig: MySQLSourceCo
   }
 }
 
+/**
+ * The PosrgreReader
+ * TODO
+ *
+ * @param session
+ * @param postgreConfig
+ */
+class PostgreSQLReader(override val session: SparkSession, postgreConfig: PostgresSQLSourceConfigEntry)
+    extends ServerBaseReader(session, postgreConfig.sentence) {
+  override def read(): DataFrame = {
+    val url =
+      s"jdbc:postgresql://${postgreConfig.host}:${postgreConfig.port}/${postgreConfig.database}"
+    val df = session.read
+      .format("jdbc")
+      .option("url", url)
+      .option("dbtable", postgreConfig.table)
+      .option("user", postgreConfig.user)
+      .option("password", postgreConfig.password)
+      .load()
+    df.createOrReplaceTempView(postgreConfig.table)
```

Comment:
How do you test the postgresql datasource? If your table is configed as `table` but the table name in sentence is `db.table`, can this sentence be executed successfully?

Reply:
I tested with
```
df.createOrReplaceTempView("table")
 session.sql("select * from db.table")
```
and got exceptions like :
```
Exception in thread "main" org.apache.spark.sql.AnalysisException: Table or view not found: `db`.`table`; line 1 pos 14;
'Project [*]
+- 'UnresolvedRelation `db`.`table`
```

So, please change the table name in config `sentence` to keep the same with config `table` in config file.